pyiron_atomistics/atomistics/structure/neighbors.py

```diff
@@ -338,53 +410,57 @@ def get_vectors(
 
 
         Returns:
-            (list/numpy.ndarray) list (if allow_ragged=True) or numpy.ndarray (otherwise) of
+            (list/numpy.ndarray) list (if mode='ragged') or numpy.ndarray (otherwise) of
                 neighbor vectors
         """
         return self._get_vectors(
             positions=positions,
             allow_ragged=allow_ragged,
+            mode=mode,
             num_neighbors=num_neighbors,
             cutoff_radius=cutoff_radius,
             width_buffer=width_buffer,
         )
 
+    @deprecate(allow_ragged="use mode instead.")
```

Comment:
Same here, let's shift the `if allow_ragged` check to `get_vectors` and just update the private function.